experiments/image-builder/image-builder.sh

```diff
@@ -43,15 +43,16 @@ VBOX_WINDOWS_ISO="${VBOX_WINDOWS_ISO:-file:/tmp/windows.iso}"
 VBOX_WINDOWS_RUNTIME="${VBOX_WINDOWS_RUNTIME:-containerd}"
 VBOX_WINDOWS_ROLES=${VBOX_WINDOWS_CUSTOM_ROLES:-cni}
 
-# Cloning the image-builder repository
```

Comment:
we should keep the repository clone here.

Reply:
You are right, that was unintentional.